src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -71,7 +71,7 @@ public abstract class OptionsBase : LearnerInputBaseWithWeight
             [Argument(ArgumentType.LastOccurenceWins, HelpText = "Init weights diameter", ShortName = "initwts", SortOrder = 140)]
             [TGUI(Label = "Initial Weights Scale", SuggestedSweeps = "0,0.1,0.5,1")]
             [TlcModule.SweepableFloatParamAttribute("InitWtsDiameter", 0.0f, 1.0f, numSteps: 5)]
-            public float InitWtsDiameter = 0;
+            public float InitialWeightsDiameter = 0;
 
             // Deprecated
```

Comment:
>// Deprecated [](start = 12, length = 13)

:)
can you double check it's true?

Reply:
looks like its used...  one of our tests `MulticlassLogisticRegressionOnnxConversionTest` even uses it by setting it to `false`

i have added a comment

---
In reply to: [260874466](https://github.com/dotnet/machinelearning/pull/2761#discussion_r260874466) [](ancestors = 260874466)

Reply:
My question is it actually used in logisticRegression code?
Across all other learners we just use `NumberOfThreads` to pass how many threads we want to use.
I just don't see reason why this one should be remain public.

---
In reply to: [260904491](https://github.com/dotnet/machinelearning/pull/2761#discussion_r260904491) [](ancestors = 260904491,260874466)

Reply:
Made internal for now.  

I see several usage of this in TestEntryPoint.cs   .  As you pointed, we should look into whether we can remove this completely  and only use `NumberOfThreads` even from EntryPoints




---
In reply to: [260905721](https://github.com/dotnet/machinelearning/pull/2761#discussion_r260905721) [](ancestors = 260905721,260904491,260874466)